CONTRIBUTING.md

```diff
@@ -1,252 +1,63 @@
-  - You must also think through anything which ought to be included but is not
-  - You must think through whether any added code could be partially combined (DRYed) with existing code
-  - You must think through any potential security issues or incentive-compatibility flaws introduced by the changes
-  - Naming must be consistent with conventions and the rest of the codebase
-  - Code must live in a reasonable location, considering dependency structures (e.g. not importing testing modules in production code, or including example code modules in production code).
-  - if you approve of the PR, you are responsible for fixing any of the issues mentioned here and more
-- If you sat down with the PR submitter and did a pairing review please note that in the `Approval`, or your PR comments.
-- If you are only making "surface level" reviews, submit any notes as `Comments` without adding a review.
+Please check out also our [Request For Maintainership](TODO:add link) process, which contains information relevant to this.
 
-### Updating Documentation
+### Architecture Decision Records (ADR)
 
-If you open a PR on ibc-go, it is mandatory to update the relevant documentation in /docs.
+When proposing an architecture decision for the ibc-go, please create an [ADR](./docs/architecture/README.md) so further discussions can be made. We are following this process so all involved parties are in agreement before any party begins coding the proposed implementation. Please use the [ADR template](./docs/architecture/adr-template.md) to scaffold any new ADR. If you would like to see some examples of how these are written refer to ibc-go's [ADRs](./docs/architecture/).
 
-## Forking
+### Participating in discussions
 
-Please note that Go requires code to live under absolute paths, which complicates forking.
-While my fork lives at `https://github.com/colin-axner/ibc-go`,
-the code should never exist at `$GOPATH/src/github.com/colin-axner/ibc-go`.
-Instead, we use `git remote` to add the fork as a new remote for the original repo,
-`$GOPATH/src/github.com/cosmos/ibc-go`, and do all the work there.
+New features or improvements are sometimes also debated in [discussions](https://github.com/cosmos/ibc-go/discussions). Sharing feedback or ideas there is very helpful for us.
 
-For instance, to create a fork and work on a branch of it, I would:
+### Submitting pull requests
 
-- Create the fork on github, using the fork button.
-- Go to the original repo checked out locally (i.e. `$GOPATH/src/github.com/cosmos/ibc-go`)
-- `git remote add fork git@github.com:colin-axner/ibc-go.git`
+Unless you feel confident your change will be accepted (trivial bug fixes, code cleanup, etc) you should first create an issue to discuss your change with us. This lets us all discuss the design and proposed implementation of your change, which helps ensure your time is well spent and that your contribution will be accepted.
```

Comment:
If an issue exists, the contributor should indicate they wish to pick up the issue